src/utils/getImageId.ts

```diff
@@ -1,16 +1,23 @@
 import { shell } from "./shell";
+const Docker = require('dockerode')
```

Comment:
To get typescript support you have to import libraries with the [`import` keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import), like with the library above.